pages/api/og.tsx

```diff
@@ -43,22 +9,21 @@ export const config = {
 export default function (req: NextRequest) {
   try {
     const { searchParams } = new URL(req.url);
-    const testUrl = new URL(req.url);
     const hasTitle = searchParams.has('title');
     const title = hasTitle
       ? searchParams.get('title')?.slice(0, 100)
       : 'My Store';
     const hasColor = searchParams.get('color')!;
     const color = hasColor ? hasColor : '000000';
-    const foreground =
-      lightOrDark(`#${color}`) == 'light' ? '000000' : 'FFFFFF';
-    const logo = searchParams.get('logo');
+    const hasForeground = searchParams.get('backgroundColor')!;
```

Comment:
Why not just save it immediate for foreground?

Heck, change 
```
    const hasColor = searchParams.get('color')!;
    const color = hasColor ? hasColor : '000000';
```

to

```
    const color = searchParams.get('color') || '000000';
```

while you're at it